backend/test/scans.test.ts

```diff
@@ -415,6 +415,5 @@ describe('run scan', () => {
       .expect(403);
 
     expect(response.body).toEqual({});
-    expect(scheduler).toHaveBeenCalledTimes(0);
```

Comment:
Why remove this?

Reply:
Wasn't my intent to test that, just a left over from copying the format of the  "invoke by globalView should fail" test above that. That test was actually testing the invoke scheduler. The test I wrote wasn't directly testing the scheduler, so I decided to remove it.